hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -416,4 +418,7 @@ private OzoneConsts() {
 
   // CRL Sequence Id
   public static final String CRL_SEQUENCE_ID_KEY = "CRL_SEQUENCE_ID";
+
+  public static final String SCM_CA_PATH = "ca";
+  public static final String SCM_CA_CERT_STORAGE_DIR = "scm";
```

Comment:
Do we differentiate the primary scm root certificate and the the scm ca certificate? I think they are all stored under {metadir}/ca/scm with CA-{certid}.pem?

Reply:
SCM Root CA location is 
{metadir}/scm/ca/keys
{metadir}/scm/ca/certs

And SCM Sub CA location is
 {metadir}/scm/sub-ca/keys 
 {metadir}/scm/sub-ca/certs